src/formio/index.ts

```diff
@@ -37,12 +39,14 @@ export * from './components';
 export type AnyComponentSchema =
   // inputs
   | TextFieldComponentSchema
+  | CheckboxComponentSchema
```

Comment:
can you order them the same way they are ordered in the form builder?

![image](https://github.com/open-formulieren/types/assets/5518550/81c28369-9f49-4837-9b8a-dd8c9aceaa23)

Reply:
Was wondering the correct order to use